pkg/cmd/kind/delete/clusters/deleteclusters.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package clusters implements the `delete` command for multiple clusters
+package clusters
+
+import (
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cmd"
+	"sigs.k8s.io/kind/pkg/errors"
+	"sigs.k8s.io/kind/pkg/log"
+)
+
+type flagpole struct {
+	Kubeconfig string
+	All        bool
+}
+
+// NewCommand returns a new cobra.Command for cluster creation
+func NewCommand(logger log.Logger, streams cmd.IOStreams) *cobra.Command {
+	flags := &flagpole{}
+	cmd := &cobra.Command{
+		Args: cobra.MinimumNArgs(0),
+		// TODO(bentheelder): more detailed usage
+		Use:   "clusters",
+		Short: "Deletes one or more clusters",
+		Long:  "Deletes a resource",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if !flags.All && len(args) == 0 {
+				return errors.New("no cluster names provided")
+			}
+
+			return deleteClusters(logger, flags, args)
+		},
+	}
+	cmd.Flags().StringVar(&flags.Kubeconfig, "kubeconfig", "", "sets kubeconfig path instead of $KUBECONFIG or $HOME/.kube/config")
```

Comment:
oooh realizing now that this is awkard if each cluster has a different kubeconfig path
oh well, something to mull in the future, that user should probably just delete them individually

Reply:
Yeah, it's a good call. I can open an issue so this is documented somewhere.